velox/dwio/parquet/tests/writer/ParquetWriterTest.cpp

```diff
@@ -748,6 +809,29 @@ TEST_F(ParquetWriterTest, parquetWriteTimestampTimeZoneWithDefault) {
   writer->close();
 };
 
+DEBUG_ONLY_TEST_F(ParquetWriterTest, parquetWriteWithArrowMemoryPool) {
+  SCOPED_TESTVALUE_SET(
+      "ArrowMemoryPool::Allocate",
+      std::function<void(const uint8_t*)>([&](const uint8_t* buffer) {
+        ASSERT_TRUE(buffer != nullptr);
```

Comment:
@rui-mo  We added a check to ensure that `buffer ` is not a nullptr, verifying that ArrowMemoryPool#Allocate is functioning correctly.